### PR TITLE
Get app list and launch app from XRPK

### PR DESCRIPTION
**Pull request by thepelkus-too**
Supports https://trello.com/c/uEopT08X/431-3-implement-ui-to-display-a-list-of-models-and-be-able-to-load-any-one-into-the-scene by adding an interface by which Unity apps can request a list of saved XRPKs and instantiate one of the saved XRPKs with a given transform by UUID.

### CODE REVIEW

**Comment by thepelkus-too** [comment]
[(3) Implement UI to display a list of models and be able to load any one into the scene](https://trello.com/c/uEopT08X/431-3-implement-ui-to-display-a-list-of-models-and-be-able-to-load-any-one-into-the-scene)

File: Runtime/Plugins/WebGL/MAMInterface.jslib [hunk 1]
```diff
@@ -1,30 +1,106 @@
...
+                })
+            );
+        },
+
+        MAMGetApps: function (receiverName, dataMethodName, errorMethodName) {
+            if (!window.AppState) return;
+
+            console.log("### in MAMGetApps")
+            console.log("### unityInstance ", unityInstance)
+            console.log("### window parent unityInstance ", window.parent.unityInstance)
```
Comment (timgcarlson) [inline Runtime/Plugins/WebGL/MAMInterface.jslib]: Do these logs still need to be here?

**Review by timgcarlson — CHANGES_REQUESTED** [review]
Do the logs still need to be in here? Everything else looks good once that question has been answered

File: Runtime/Plugins/WebGL/MAMInterface.jslib (hunk 1 above)
Comment (thepelkus-too) [reply Runtime/Plugins/WebGL/MAMInterface.jslib]: Oh! No. They are gross. I will leave them on the curb.

Comment (thepelkus-too) [reply Runtime/Plugins/WebGL/MAMInterface.jslib]: Updated!

**Review by timgcarlson — APPROVED** [review]

**Pull request merged**